### DISCUSSION TITLE
'Connection attempt failed' error

### DISCUSSION THREAD

**Discussion by Aks-Tiaab** [pspete/psPAS#511] [Q&A]
Hello everyone,

I hope this is not an issue that was already answered. I have this error when running the command `New-PASSession -Credential $Admin -BaseURI *pvwa_uri*`.
```
Invoke-WebRequest:
Line |
 216 |  … $APIResponse = Invoke-WebRequest @PSBoundParameters -ErrorAction Stop
     |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
```
I can't figure out the reason why, and I already tested some stuff:
- the PVWA works fine on the browser (I can connect with the same credentials used in the command)
- I can ping the PVWA server and it responds back
- my DNS server knows the *pvwa_uri* real IP address (I am working on VMs, in a lab environment)

Does someone already had this issue or can help me find the reason of it?

Thanks in advance,
Aks

**Comment by pspete** [comment]
Its not a message i recall ever seeing @Aks-Tiaab  - hopefully someone will be able to chime in.
